pandas/_libs/groupby.pxd

```diff
@@ -0,0 +1,6 @@
+cdef enum InterpolationEnumType:
```

Comment:
IIRC we reverted using enum in other versions (because this would require an additional install in py2)?

Reply:
Hmm we have something similar in algos:

https://github.com/pandas-dev/pandas/blob/decc8ce5601c32dc2313f0d2b1d611164262cae9/pandas/_libs/algos.pxd#L15

Do you know where that issue would have popped up?

Reply:
ok great.